.idea/dataSources.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are the .idea files supposed to be checked in?  I don't know what they are or how they work.

Are they necessary for the program to run now?  How are they referenced from within the code?

Reply:
No, not necessary. Files left over from PhpStorm. With the exception of the .gitignore file, all the dot files and directories are not necessary for the code to run.

Will address.

Reply:
OK

Reply:
.gitignore now has a 
./.idea/
line that should ignore the .idea directory